asc-miniapp/.env.example

```
@@ -0,0 +1,12 @@
+# MiniKit (OnchainKit)
+NEXT_PUBLIC_ONCHAINKIT_API_KEY=your_cdp_api_key_here
+NEXT_PUBLIC_URL=http://localhost:3001
+NEXT_PUBLIC_ONCHAINKIT_PROJECT_NAME=Ascencia
+
+# Supabase (same as asc-frontend)
+NEXT_PUBLIC_SUPABASE_URL=https://znpncymanbvwgopftscm.supabase.co
+NEXT_PUBLIC_SUPABASE_ANON_KEY=your_anon_key_here
+
+# Contract (when ready)
+NEXT_PUBLIC_REPUTATION_CONTRACT=0x...
+NEXT_PUBLIC_CHAIN_ID=84532
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)